File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        map = defaultdict(int)
        for i in nums:
            map[i] += 1
        h = []
        for i in map.keys():
            h.append((-1 * map.get(i),i))
        heapq.heapify(h)
        print(h)
        ans=[]
        for i in range(k):
            ans.append(heapq.heappop(h)[1])
        return ans
        

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
  public int removeDuplicates(int[] nums) {
        int l = 0, ctr = 0;

        for (int r = 1; r < nums.length; r++) {
            if (nums[l] != nums[r]) {
                nums[++ctr] = nums[r];
                l = r;
            }
        }

        return ctr +1;
    }
}